test/fixtures/variables.css

```diff
@@ -0,0 +1,8 @@
+:root {
+  --some-var: #fff;
+  --some-other-var: var(--some-other-var);
```

Comment:
Should the value here be `var(--some-var)`? Just want to be sure I understand what this is testing.